src/commands/BootstrapCommand.js

```diff
@@ -49,24 +50,24 @@ export default class BootstrapCommand extends Command {
       const linkSrc = dependency.location;
       const linkDest = path.join(pkg.nodeModulesLocation, dependency.name);
 
-      this.createLinkedDependency(linkSrc, linkDest, dependency.name, done);
+      this.createLinkedDependency(linkSrc, linkDest, dependency.name, (err) => {
+        if (!err && dependency._package.bin) {
```

Comment:
`bin` can be [an object](https://docs.npmjs.com/files/package.json#bin).  Should probably support that here.